server/core/nats2kafka_test.go

```diff
@@ -42,13 +42,13 @@ func TestSimpleSendOnNatsReceiveOnKafka(t *testing.T) {
 
 	tbs.Bridge.checkConnections()
 
-	err = tbs.NC.Publish("test", []byte(msg))
-	require.NoError(t, err)
-
 	reader := tbs.CreateReader(topic, 5000)
 	defer reader.Close()
 
-	_, data, err := tbs.GetMessageFromKafka(reader, 5000)
+	err = tbs.NC.Publish("test", []byte(msg))
+	require.NoError(t, err)
+
+	_, data, err := tbs.GetMessageFromKafka(reader, 10000)
```

Comment:
...could create a constant for the timeout duration.

Reply:
Hm. Yeah, thinking about what to do here. Changing the timeouts actually didn't help, so I'll probably revert this change. I'm wondering if there are some more fundamental tweaks we need. The tests failing on CI also fail on my laptop, if I run the tests enough times. CI is really good at exposing them though! :smile: